app/src/ui_stream.c

```diff
@@ -304,8 +317,11 @@ find_icon(tz_ui_icon_t icon)
     // clang-format on
 }
 
+#define COLOR_BLACK 0xFFFFFF
+#define COLOR_WHITE 0x000000
+
 static void
-redisplay_bnp(void)
+display_init(bagl_element_t init[], uint8_t icon_pos)
```

Comment:
```suggestion
display_init(bagl_element_t init[UI_INIT_ARRAY_LEN], uint8_t icon_pos)
```

Otherwise, you technically don't know the size of the array within the function?

Reply:
Done.